versioning/src/main/scala/org/typelevel/sbt/TypelevelVersioningPlugin.scala

```diff
@@ -76,10 +76,10 @@ object TypelevelVersioningPlugin extends AutoPlugin {
           .flatMap { previous =>
             if (previous > baseV)
               sys.error(s"Your tlBaseVersion $baseV is behind the latest tag $previous")
-            else if (baseV.isSameSeries(previous))
-              Some(previous)
+            else if (!baseV.isSameSeries(previous))
+              sys.error(s"Your tlBaseVersion $baseV has to be in the same series with latest tag $previous")
```

Comment:
I'm not sure if this is what we want to check 🤔  IIUC this is checking if the base version is not in the same series `x.y` as the previous release, to throw an error. For example, if the base version is `0.5` and the previous release was `v0.4.3`. Actually, this is exactly the situation on the main branch of sbt-typelevel :)

AFAICT there's nothing wrong with this situation: it is normal for the base version to be ahead of the version of the last release, because it indicates you have started working on the next version of your project. You can test this by bumping the base version in `build.sbt` to 0.5 and reloading sbt.

I think this is why when your PR originally targeted the main branch it was failing:
https://github.com/typelevel/sbt-typelevel/runs/5047968979?check_suite_focus=true#step:5:47

An example of the situation that we do want to prevent is if the current version is `0.4.4` but the base version is `0.5`. The reason this is bad is because if the developed bumped the base version to `0.5` that must mean that their project is no longer compatible with the `0.4` series. In which case, if the current version is `0.4.4` then there is something wrong.

Let me know if that makes sense :)

Reply:
Makes sense :)